modules/ROOT/pages/_partials/rn-new-and-changed-ref.adoc

```diff
@@ -1,6 +1,8 @@
 [id='rn-new-and-changed-ref']
 = New and changed features
 
+* With this release, RHMI only supports TLS 1.2
```

Comment:
RHMI only supports TLS 1.2 in this release.


Just a suggestion, but I don't think there is much of a difference

Reply:
actually, you've just made me realise that we shouldn't say "RHMI".  Eeeek.